stdlib/public/Concurrency/PartialAsyncTask.swift

```diff
@@ -47,6 +47,16 @@ public struct UnsafeThrowingContinuation<T> {
   public func resume(returning: __owned T)
   @_silgen_name("swift_continuation_throwingResumeWithError")
   public func resume(throwing: __owned Error)
+
+  @_alwaysEmitIntoClient
+  public func resume(with result: Result<T, Error>) {
```

Comment:
This could be generic over the `Error` argument, since we can type-erase any specific error type in a `Result` into an untyped `Error` to throw through the originating task.

Reply:
By this do you mean, have it be `func resume<E>(with result: Result<T, E>)` ?

Reply:
Yeah.

Reply:
Done. I added a constraint on `E` to be an `Error`.